zwavefibaro/zwavefibaro.pro

```diff
@@ -0,0 +1,12 @@
+include(../plugins.pri)
+
+SOURCES += \
+    ../common/zwaveintegrationplugin.cpp \
```

Comment:
since we planed to create shared plugin lib for zigbee, would it make sense to provide that for zwave right from the beginning?